EXAMPLES.md

```diff
@@ -0,0 +1,98 @@
+# Very common words; (in lists: All, Default, Common, Very Common)
```

Comment:
@jacksonrayhamilton For better hierarchy and since h1's are rather oversized, I would recommend having the only `#` heading (h1) in this file be `# Examples`, and then convert all the word category headings (like this one) to h2 (`##`).

Reply:
Good idea.

Reply:
@jacksonrayhamilton Saw that you updated this. Resolving this thread.